src/ru/skypro/Main.java

```diff
@@ -4,5 +4,34 @@
 
     public static void main(String[] args) {
 	// write your code here
+        System.out.println( "Task 1" );
+        int cycle = 0;
+        while (cycle < 10) {
+            cycle = cycle + 1;
+            System.out.println(cycle);
```

Comment:
для вывода значений в строку нужно использовать print
```suggestion
            System.out.print(cycle);
```